src/session/session_dhandle.c

```diff
@@ -306,6 +306,16 @@ __wt_session_release_btree(WT_SESSION_IMPL *session)
 }
 
 /*
+ * __wt_session_release_btree --
+ *	Unlock a btree handle.
+ */
+int
+__wt_session_release_btree(WT_SESSION_IMPL *session)
```

Comment:
Would `__wt_session_release_btree` and `__wt_session_get_btree` ideally go away?